src/main/resources/config/application.yaml

```diff
@@ -2,13 +2,6 @@ spring:
   application:
     name: config-notification-server
 
-  profiles:
-    active: default,local
-
-  rabbitmq:
```

Comment:
Just for my flacky knowledge in these areas : how this notification server is no more bound to rabbitmq ?

Reply:
it's still bound to rabbitmq: see common conf in [here](https://github.com/powsybl/powsybl-ws-commons/blob/main/src/main/resources/application.yaml)